src/get-throw.js

```diff
@@ -0,0 +1,11 @@
+import getThrowFromNumber from '../src/throw-from-number.js';
+
+
```

Comment:
You have some unnecessary white space throughout your files. I'm taking a point off for clean/spaced code, but if you resubmit and delete unnecessary white space I'll regrade and give you that point back!